2017-12-03T21:00:40.242+0900 I CONTROL  [initandlisten] MongoDB starting : pid=9419 port=37018 dbpath=/data/shard0/rs1 64-bit host=kris-ThinkPad-X1-Carbon-5th
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten] db version v3.4.10
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten] git version: 078f28920cb24de0dd479b5ea6c66c644f6326e9
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2g  1 Mar 2016
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten] allocator: tcmalloc
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten] modules: none
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten] build environment:
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten]     distmod: ubuntu1604
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten]     distarch: x86_64
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten]     target_arch: x86_64
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten] options: { net: { port: 37018 }, processManagement: { fork: true }, replication: { replSet: "s0" }, sharding: { clusterRole: "shardsvr" }, storage: { dbPath: "/data/shard0/rs1" }, systemLog: { destination: "file", path: "s0-r1.log" } }
2017-12-03T21:00:40.266+0900 I STORAGE  [initandlisten] 
2017-12-03T21:00:40.266+0900 I STORAGE  [initandlisten] ** WARNING: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine
2017-12-03T21:00:40.266+0900 I STORAGE  [initandlisten] **          See http://dochub.mongodb.org/core/prodnotes-filesystem
2017-12-03T21:00:40.266+0900 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=7269M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),checkpoint=(wait=60,log_size=2GB),statistics_log=(wait=0),
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] ** WARNING: You are running this process as the root user, which is not recommended.
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] ** WARNING: /sys/kernel/mm/transparent_hugepage/enabled is 'always'.
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] **        We suggest setting it to 'never'
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:40.354+0900 W SHARDING [initandlisten] Started with --shardsvr, but no shardIdentity document was found on disk in admin.system.version. This most likely means this server has not yet been added to a sharded cluster.
2017-12-03T21:00:40.378+0900 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/shard0/rs1/diagnostic.data'
2017-12-03T21:00:40.401+0900 I REPL     [initandlisten] Did not find local voted for document at startup.
2017-12-03T21:00:40.401+0900 I REPL     [initandlisten] Did not find local replica set configuration document at startup;  NoMatchingDocument: Did not find replica set configuration document in local.system.replset
2017-12-03T21:00:40.401+0900 I NETWORK  [thread1] waiting for connections on port 37018
2017-12-03T21:00:45.646+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36582 #1 (1 connection now open)
2017-12-03T21:00:45.647+0900 I -        [conn1] end connection 127.0.0.1:36582 (1 connection now open)
2017-12-03T21:00:45.647+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36586 #2 (1 connection now open)
2017-12-03T21:00:45.647+0900 I NETWORK  [conn2] received client metadata from 127.0.0.1:36586 conn2: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:00:45.647+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:37017
2017-12-03T21:00:45.648+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:37017, took 1ms (1 connections now open to localhost:37017)
2017-12-03T21:00:47.651+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36600 #3 (2 connections now open)
2017-12-03T21:00:47.652+0900 I -        [conn3] end connection 127.0.0.1:36600 (2 connections now open)
2017-12-03T21:00:47.681+0900 I REPL     [replExecDBWorker-0] Starting replication storage threads
2017-12-03T21:00:47.760+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36604 #4 (2 connections now open)
2017-12-03T21:00:47.761+0900 I NETWORK  [conn4] received client metadata from 127.0.0.1:36604 conn4: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:00:47.762+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:37019
2017-12-03T21:00:47.762+0900 I REPL     [replication-0] Starting initial sync (attempt 1 of 10)
2017-12-03T21:00:47.762+0900 I REPL     [ReplicationExecutor] New replica set config in use: { _id: "s0", version: 1, protocolVersion: 1, members: [ { _id: 0, host: "localhost:37017", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 }, { _id: 1, host: "localhost:37018", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 }, { _id: 2, host: "localhost:37019", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 } ], settings: { chainingAllowed: true, heartbeatIntervalMillis: 2000, heartbeatTimeoutSecs: 10, electionTimeoutMillis: 10000, catchUpTimeoutMillis: 60000, getLastErrorModes: {}, getLastErrorDefaults: { w: 1, wtimeout: 0 }, replicaSetId: ObjectId('5a23e76d05bd90c23620c9e4') } }
2017-12-03T21:00:47.762+0900 I REPL     [ReplicationExecutor] This node is localhost:37018 in the config
2017-12-03T21:00:47.762+0900 I REPL     [ReplicationExecutor] transition to STARTUP2
2017-12-03T21:00:47.763+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:37019
2017-12-03T21:00:47.763+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:37019, took 1ms (2 connections now open to localhost:37019)
2017-12-03T21:00:47.764+0900 I REPL     [ReplicationExecutor] Member localhost:37017 is now in state SECONDARY
2017-12-03T21:00:47.764+0900 I REPL     [ReplicationExecutor] Member localhost:37019 is now in state STARTUP2
2017-12-03T21:00:47.764+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:37019, took 1ms (2 connections now open to localhost:37019)
2017-12-03T21:00:47.805+0900 I REPL     [replication-0] sync source candidate: localhost:37017
2017-12-03T21:00:47.805+0900 I STORAGE  [replication-0] dropAllDatabasesExceptLocal 1
2017-12-03T21:00:47.805+0900 I REPL     [replication-0] ******
2017-12-03T21:00:47.805+0900 I REPL     [replication-0] creating replication oplog of size: 6161MB...
2017-12-03T21:00:47.825+0900 I STORAGE  [replication-0] Starting WiredTigerRecordStoreThread local.oplog.rs
2017-12-03T21:00:47.825+0900 I STORAGE  [replication-0] The size storer reports that the oplog contains 0 records totaling to 0 bytes
2017-12-03T21:00:47.825+0900 I STORAGE  [replication-0] Scanning the oplog to determine where to place markers for truncation
2017-12-03T21:00:47.922+0900 I REPL     [replication-0] ******
2017-12-03T21:00:47.922+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-03T21:00:47.923+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 1ms (1 connections now open to localhost:37017)
2017-12-03T21:00:47.924+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-03T21:00:47.925+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 1ms (2 connections now open to localhost:37017)
2017-12-03T21:00:47.926+0900 I REPL     [replication-1] No need to apply operations. (currently at { : Timestamp 1512302445000|1 })
2017-12-03T21:00:47.926+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending connection to host localhost:37017 due to bad connection status; 1 connections to that host remain open
2017-12-03T21:00:47.926+0900 I REPL     [replication-1] Finished fetching oplog during initial sync: CallbackCanceled: Callback canceled. Last fetched optime and hash: { ts: Timestamp 1512302445000|1, t: -1 }[-3972036409932239766]
2017-12-03T21:00:47.926+0900 I REPL     [replication-1] Initial sync attempt finishing up.
2017-12-03T21:00:47.926+0900 I REPL     [replication-1] Initial Sync Attempt Statistics: { failedInitialSyncAttempts: 0, maxFailedInitialSyncAttempts: 10, initialSyncStart: new Date(1512302447762), initialSyncAttempts: [], fetchedMissingDocs: 0, appliedOps: 0, initialSyncOplogStart: Timestamp 1512302445000|1, initialSyncOplogEnd: Timestamp 1512302445000|1, databases: { databasesCloned: 0 } }
2017-12-03T21:00:47.930+0900 I REPL     [replication-0] initial sync done; took 0s.
2017-12-03T21:00:47.930+0900 I REPL     [replication-0] Starting replication fetcher thread
2017-12-03T21:00:47.931+0900 I REPL     [replication-0] Starting replication applier thread
2017-12-03T21:00:47.931+0900 I REPL     [replication-0] Starting replication reporter thread
2017-12-03T21:00:47.931+0900 I REPL     [rsSync] transition to RECOVERING
2017-12-03T21:00:47.931+0900 I REPL     [rsBackgroundSync] could not find member to sync from
2017-12-03T21:00:47.931+0900 I REPL     [ReplicationExecutor] Member localhost:37019 is now in state SECONDARY
2017-12-03T21:00:47.931+0900 I REPL     [rsSync] transition to SECONDARY
2017-12-03T21:00:57.933+0900 I REPL     [ReplicationExecutor] Member localhost:37017 is now in state PRIMARY
2017-12-03T21:00:58.933+0900 I REPL     [rsBackgroundSync] sync source candidate: localhost:37017
2017-12-03T21:00:58.936+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-03T21:00:58.937+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 1ms (2 connections now open to localhost:37017)
2017-12-03T21:01:47.764+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending idle connection to host localhost:37019 because the pool meets constraints; 1 connections to that host remain open
2017-12-03T21:02:10.569+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36776 #5 (3 connections now open)
2017-12-03T21:02:10.569+0900 I NETWORK  [conn5] received client metadata from 127.0.0.1:36776 conn5: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.648+0900 I INDEX    [repl writer worker 1] build index on: admin.system.version properties: { v: 2, key: { version: 1 }, name: "incompatible_with_version_32", ns: "admin.system.version" }
2017-12-03T21:02:10.648+0900 I INDEX    [repl writer worker 1] 	 building index using bulk method; build may temporarily use up to 500 megabytes of RAM
2017-12-03T21:02:10.650+0900 I INDEX    [repl writer worker 1] build index done.  scanned 0 total records. 0 secs
2017-12-03T21:02:10.653+0900 I COMMAND  [repl writer worker 4] setting featureCompatibilityVersion to 3.4
2017-12-03T21:02:10.659+0900 I SHARDING [repl writer worker 8] initializing sharding state with: { _id: "shardIdentity", configsvrConnectionString: "csReplSet/localhost:57040,localhost:57041,localhost:57042", shardName: "s0", clusterId: ObjectId('5a23e786bf51d32cbcc946a4') }
2017-12-03T21:02:10.659+0900 I SHARDING [repl writer worker 8] first cluster operation detected, adding sharding hook to enable versioning and authentication to remote servers
2017-12-03T21:02:10.660+0900 I NETWORK  [repl writer worker 8] Starting new replica set monitor for csReplSet/localhost:57040,localhost:57041,localhost:57042
2017-12-03T21:02:10.661+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57040 (1 connections now open to localhost:57040 with a 5 second timeout)
2017-12-03T21:02:10.661+0900 I SHARDING [thread2] creating distributed lock ping thread for process kris-ThinkPad-X1-Carbon-5th:37018:1512302530:-6460390088971249845 (sleeping for 30000ms)
2017-12-03T21:02:10.662+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-03T21:02:10.662+0900 I NETWORK  [replSetDistLockPinger] Successfully connected to localhost:57042 (1 connections now open to localhost:57042 with a 5 second timeout)
2017-12-03T21:02:10.662+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57041 (1 connections now open to localhost:57041 with a 5 second timeout)
2017-12-03T21:02:10.662+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-03T21:02:10.662+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 0ms (2 connections now open to localhost:57040)
2017-12-03T21:02:10.663+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (2 connections now open to localhost:57040)
2017-12-03T21:02:10.672+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: LockStateChangeFailed: findAndModify query predicate didn't match any lock document
2017-12-03T21:02:10.740+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36824 #6 (4 connections now open)
2017-12-03T21:02:10.740+0900 I NETWORK  [conn6] received client metadata from 127.0.0.1:36824 conn6: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.821+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36850 #7 (5 connections now open)
2017-12-03T21:02:10.822+0900 I NETWORK  [conn7] received client metadata from 127.0.0.1:36850 conn7: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.837+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36876 #8 (6 connections now open)
2017-12-03T21:02:10.837+0900 I NETWORK  [conn8] received client metadata from 127.0.0.1:36876 conn8: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.856+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36894 #9 (7 connections now open)
2017-12-03T21:02:10.856+0900 I NETWORK  [conn9] received client metadata from 127.0.0.1:36894 conn9: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.929+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36946 #10 (8 connections now open)
2017-12-03T21:02:10.929+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36948 #11 (9 connections now open)
2017-12-03T21:02:10.929+0900 I NETWORK  [conn10] received client metadata from 127.0.0.1:36946 conn10: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.929+0900 I NETWORK  [conn11] received client metadata from 127.0.0.1:36948 conn11: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.991+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36974 #12 (10 connections now open)
2017-12-03T21:02:10.991+0900 I NETWORK  [conn12] received client metadata from 127.0.0.1:36974 conn12: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:11.140+0900 I INDEX    [repl writer worker 10] build index on: school.students properties: { v: 2, key: { student_id: 1.0 }, name: "student_id_1", ns: "school.students" }
2017-12-03T21:02:11.140+0900 I INDEX    [repl writer worker 10] 	 building index using bulk method; build may temporarily use up to 500 megabytes of RAM
2017-12-03T21:02:11.141+0900 I INDEX    [repl writer worker 10] build index done.  scanned 0 total records. 0 secs
2017-12-03T21:02:11.200+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36992 #13 (11 connections now open)
2017-12-03T21:02:11.201+0900 I NETWORK  [conn13] received client metadata from 127.0.0.1:36992 conn13: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:27.450+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37024 #14 (12 connections now open)
2017-12-03T21:02:27.451+0900 I NETWORK  [conn14] received client metadata from 127.0.0.1:37024 conn14: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:27.606+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37042 #15 (13 connections now open)
2017-12-03T21:02:27.606+0900 I NETWORK  [conn15] received client metadata from 127.0.0.1:37042 conn15: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:40.664+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-03T21:02:40.666+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-03T21:02:40.667+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37062 #16 (14 connections now open)
2017-12-03T21:02:40.667+0900 I NETWORK  [shard registry reload] Starting new replica set monitor for s0/localhost:37017,localhost:37018,localhost:37019
2017-12-03T21:02:40.667+0900 I NETWORK  [shard registry reload] Starting new replica set monitor for s1/localhost:47017,localhost:47018,localhost:47019
2017-12-03T21:02:40.668+0900 I NETWORK  [shard registry reload] Starting new replica set monitor for s2/localhost:57017,localhost:57018,localhost:57019
2017-12-03T21:02:40.667+0900 I NETWORK  [conn16] received client metadata from 127.0.0.1:37062 conn16: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:40.670+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37017 (1 connections now open to localhost:37017 with a 5 second timeout)
2017-12-03T21:02:40.671+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37068 #17 (15 connections now open)
2017-12-03T21:02:40.672+0900 I NETWORK  [conn17] received client metadata from 127.0.0.1:37068 conn17: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:40.672+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37018 (1 connections now open to localhost:37018 with a 5 second timeout)
2017-12-03T21:02:40.675+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37019 (1 connections now open to localhost:37019 with a 5 second timeout)
2017-12-03T21:02:40.679+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47017 (1 connections now open to localhost:47017 with a 5 second timeout)
2017-12-03T21:02:40.681+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47018 (1 connections now open to localhost:47018 with a 5 second timeout)
2017-12-03T21:02:40.683+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47019 (1 connections now open to localhost:47019 with a 5 second timeout)
2017-12-03T21:02:40.684+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57017 (1 connections now open to localhost:57017 with a 5 second timeout)
2017-12-03T21:02:40.685+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57018 (1 connections now open to localhost:57018 with a 5 second timeout)
2017-12-03T21:02:40.686+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57019 (1 connections now open to localhost:57019 with a 5 second timeout)
2017-12-03T21:03:10.663+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Ending idle connection to host localhost:57040 because the pool meets constraints; 1 connections to that host remain open
2017-12-03T21:08:40.678+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T21:08:40.680+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-03T21:12:40.690+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-03T21:12:40.691+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-03T21:36:49.550+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-03T21:36:49.558+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 8ms (3 connections now open to localhost:37017)
2017-12-03T21:38:53.578+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 2 connections to that host remain open
2017-12-03T21:41:10.760+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-03T21:41:10.760+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 0ms (1 connections now open to localhost:57042)
2017-12-03T21:51:40.784+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T21:51:40.785+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-03T22:28:10.868+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T22:28:10.871+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 4ms (1 connections now open to localhost:57041)
2017-12-03T23:04:10.945+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T23:04:10.947+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 3ms (1 connections now open to localhost:57041)
2017-12-03T23:10:40.960+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T23:10:40.962+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T00:01:11.089+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T00:01:11.091+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T00:14:11.120+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T00:14:11.122+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T01:24:41.293+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T01:24:41.294+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T01:29:11.304+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T01:29:11.306+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T02:19:41.439+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T02:19:41.441+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T02:37:41.489+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T02:37:41.491+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T02:43:11.504+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T02:43:11.506+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T03:13:11.584+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T03:13:11.586+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T03:55:11.692+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T03:55:11.694+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T04:08:41.726+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T04:08:41.727+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-04T04:09:41.729+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T04:09:41.730+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T04:14:41.741+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T04:14:41.743+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T05:46:41.981+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T05:46:41.982+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T05:58:42.030+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T05:58:42.031+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T06:19:23.833+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-04T06:19:23.834+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 1ms (3 connections now open to localhost:37017)
2017-12-04T06:20:23.834+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T06:36:42.117+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T06:36:42.118+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T06:49:12.149+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T06:49:12.151+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T06:51:23.900+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-04T06:51:23.901+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 1ms (3 connections now open to localhost:37017)
2017-12-04T06:52:23.900+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T08:13:42.369+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T08:13:42.373+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 4ms (1 connections now open to localhost:57042)
2017-12-04T15:41:54.175+0900 I REPL     [ReplicationExecutor] Starting an election, since we've seen no PRIMARY in the past 10000ms
2017-12-04T15:41:54.190+0900 I REPL     [ReplicationExecutor] conducting a dry run election to see if we could be elected
2017-12-04T15:41:54.191+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:37017
2017-12-04T15:41:54.191+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:37019
2017-12-04T15:41:54.194+0900 I REPL     [ReplicationExecutor] VoteRequester(term 1 dry run) received a yes vote from localhost:37019; response message: { term: 1, voteGranted: true, reason: "", ok: 1.0 }
2017-12-04T15:41:54.194+0900 I REPL     [ReplicationExecutor] dry election run succeeded, running for election
2017-12-04T15:41:54.196+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:37017
2017-12-04T15:41:54.198+0900 I REPL     [ReplicationExecutor] VoteRequester(term 2) received a yes vote from localhost:37019; response message: { term: 2, voteGranted: true, reason: "", ok: 1.0 }
2017-12-04T15:41:54.198+0900 I REPL     [ReplicationExecutor] election succeeded, assuming primary role in term 2
2017-12-04T15:41:54.198+0900 I REPL     [ReplicationExecutor] transition to PRIMARY
2017-12-04T15:41:54.198+0900 I REPL     [ReplicationExecutor] Entering primary catch-up mode.
2017-12-04T15:41:54.198+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending connection to host localhost:37017 due to bad connection status; 2 connections to that host remain open
2017-12-04T15:41:54.199+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:37017
2017-12-04T15:41:54.211+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57040 as failed :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T15:41:54.211+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T15:41:54.212+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:37019, took 21ms (2 connections now open to localhost:37019)
2017-12-04T15:41:54.215+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:37017, took 24ms (3 connections now open to localhost:37017)
2017-12-04T15:41:54.215+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:37017, took 19ms (3 connections now open to localhost:37017)
2017-12-04T15:41:54.216+0900 I REPL     [ReplicationExecutor] Member localhost:37017 is now in state SECONDARY
2017-12-04T15:41:54.216+0900 I REPL     [ReplicationExecutor] Caught up to the latest optime known via heartbeats after becoming primary.
2017-12-04T15:41:54.216+0900 I REPL     [ReplicationExecutor] Exited primary catch-up mode.
2017-12-04T15:41:54.217+0900 I REPL     [rsBackgroundSync] Replication producer stopped after oplog fetcher finished returning a batch from our sync source.  Abandoning this batch of oplog entries and re-evaluating our sync source.
2017-12-04T15:41:54.217+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:37017, took 18ms (3 connections now open to localhost:37017)
2017-12-04T15:41:55.216+0900 I REPL     [rsSync] transition to primary complete; database writes are now permitted
2017-12-04T15:41:57.019+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 1 connections to that host remain open
2017-12-04T15:41:57.019+0900 I REPL     [SyncSourceFeedback] SyncSourceFeedback error sending update to localhost:37017: InvalidSyncSource: Sync source was cleared. Was localhost:37017
2017-12-04T15:41:57.178+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:40968 #18 (16 connections now open)
2017-12-04T15:41:57.179+0900 I NETWORK  [conn18] received client metadata from 127.0.0.1:40968 conn18: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:41:57.182+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:40970 #19 (17 connections now open)
2017-12-04T15:41:57.183+0900 I NETWORK  [conn19] received client metadata from 127.0.0.1:40970 conn19: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:41:59.216+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:40996 #20 (18 connections now open)
2017-12-04T15:41:59.216+0900 I NETWORK  [conn20] received client metadata from 127.0.0.1:40996 conn20: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:41:59.217+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:40998 #21 (19 connections now open)
2017-12-04T15:41:59.217+0900 I NETWORK  [conn21] received client metadata from 127.0.0.1:40998 conn21: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:42:07.237+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:41006 #22 (20 connections now open)
2017-12-04T15:42:07.237+0900 I NETWORK  [conn22] received client metadata from 127.0.0.1:41006 conn22: { driver: { name: "NetworkInterfaceASIO-ShardRegistry", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:42:10.310+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T15:42:10.310+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 0ms (1 connections now open to localhost:57042)
2017-12-04T15:42:10.586+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:57040)
2017-12-04T15:42:10.588+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57040 (1 connections now open to localhost:57040 with a 5 second timeout)
2017-12-04T15:42:10.590+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:37017)
2017-12-04T15:42:10.592+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37017 (1 connections now open to localhost:37017 with a 5 second timeout)
2017-12-04T15:42:10.593+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:47017)
2017-12-04T15:42:10.595+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47017 (1 connections now open to localhost:47017 with a 5 second timeout)
2017-12-04T15:42:10.596+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:57017)
2017-12-04T15:42:10.598+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57017 (1 connections now open to localhost:57017 with a 5 second timeout)
2017-12-04T15:42:24.212+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T15:42:24.214+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T15:42:54.201+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending idle connection to host localhost:37019 because the pool meets constraints; 1 connections to that host remain open
2017-12-04T15:42:54.215+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T15:42:54.218+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 1 connections to that host remain open
2017-12-04T16:21:41.131+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T16:21:41.135+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 4ms (1 connections now open to localhost:57042)
2017-12-04T16:45:25.185+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58774 #23 (21 connections now open)
2017-12-04T16:45:25.185+0900 I NETWORK  [conn23] received client metadata from 127.0.0.1:58774 conn23: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:25.192+0900 I REPL     [conn23] stepping down from primary, because a new term has begun: 3
2017-12-04T16:45:25.192+0900 I REPL     [replExecDBWorker-2] transition to SECONDARY
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] legacy transport layer closing all connections
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 23
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 22
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 21
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 20
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 19
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 18
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 4
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 2
2017-12-04T16:45:25.192+0900 I -        [conn16] end connection 127.0.0.1:37062 (21 connections now open)
2017-12-04T16:45:25.192+0900 I -        [conn10] end connection 127.0.0.1:36946 (20 connections now open)
2017-12-04T16:45:25.192+0900 I -        [conn9] end connection 127.0.0.1:36894 (19 connections now open)
2017-12-04T16:45:25.192+0900 I -        [conn14] end connection 127.0.0.1:37024 (18 connections now open)
2017-12-04T16:45:25.192+0900 I -        [conn15] end connection 127.0.0.1:37042 (17 connections now open)
2017-12-04T16:45:25.193+0900 I -        [conn17] end connection 127.0.0.1:37068 (16 connections now open)
2017-12-04T16:45:25.193+0900 I -        [conn11] end connection 127.0.0.1:36948 (15 connections now open)
2017-12-04T16:45:25.193+0900 I -        [conn7] end connection 127.0.0.1:36850 (14 connections now open)
2017-12-04T16:45:25.197+0900 I -        [conn13] end connection 127.0.0.1:36992 (13 connections now open)
2017-12-04T16:45:25.197+0900 I -        [conn12] end connection 127.0.0.1:36974 (12 connections now open)
2017-12-04T16:45:25.197+0900 I -        [conn8] end connection 127.0.0.1:36876 (11 connections now open)
2017-12-04T16:45:25.197+0900 I -        [conn6] end connection 127.0.0.1:36824 (10 connections now open)
2017-12-04T16:45:25.197+0900 I -        [conn5] end connection 127.0.0.1:36776 (9 connections now open)
2017-12-04T16:45:25.203+0900 I COMMAND  [conn20] command local.oplog.rs command: getMore { getMore: 38453556961, collection: "oplog.rs", maxTimeMS: 5000, term: 2, lastKnownCommittedOpTime: { ts: Timestamp 1512372445000|1, t: 2 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512342829000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 2, readConcern: { afterOpTime: { ts: Timestamp 1512342829000|1, t: 1 } } } planSummary: COLLSCAN cursorid:38453556961 keysExamined:0 docsExamined:0 numYields:0 nreturned:0 reslen:563 locks:{ Global: { acquireCount: { r: 4 }, acquireWaitCount: { r: 1 }, timeAcquiringMicros: { r: 320 } }, Database: { acquireCount: { r: 2 } }, oplog: { acquireCount: { r: 2 } } } protocol:op_command 1074905ms
2017-12-04T16:45:25.204+0900 I -        [conn4] end connection 127.0.0.1:36604 (8 connections now open)
2017-12-04T16:45:25.204+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58790 #24 (8 connections now open)
2017-12-04T16:45:25.205+0900 I NETWORK  [conn24] received client metadata from 127.0.0.1:58790 conn24: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:25.205+0900 I -        [conn23] end connection 127.0.0.1:58774 (8 connections now open)
2017-12-04T16:45:25.208+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 1094 secs, remote host 127.0.0.1:37018)
2017-12-04T16:45:25.208+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 1094 secs, remote host 127.0.0.1:47019)
2017-12-04T16:45:25.208+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 1094 secs, remote host 127.0.0.1:57019)
2017-12-04T16:45:25.208+0900 I COMMAND  [conn18] command local.oplog.rs command: getMore { getMore: 35231168519, collection: "oplog.rs", maxTimeMS: 5000, term: 2, lastKnownCommittedOpTime: { ts: Timestamp 1512372445000|1, t: 2 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512342829000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 2, readConcern: { afterOpTime: { ts: Timestamp 1512342829000|1, t: 1 } } } planSummary: COLLSCAN cursorid:35231168519 keysExamined:0 docsExamined:0 numYields:0 nreturned:0 reslen:563 locks:{ Global: { acquireCount: { r: 4 }, acquireWaitCount: { r: 1 }, timeAcquiringMicros: { r: 5332 } }, Database: { acquireCount: { r: 2 } }, oplog: { acquireCount: { r: 2 } } } protocol:op_command 1074910ms
2017-12-04T16:45:25.208+0900 I -        [conn18] end connection 127.0.0.1:40968 (7 connections now open)
2017-12-04T16:45:25.213+0900 I REPL     [ReplicationExecutor] Member localhost:37017 is now in state PRIMARY
2017-12-04T16:45:25.256+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57041 as failed :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T16:45:25.256+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T16:45:25.892+0900 I -        [conn21] end connection 127.0.0.1:40998 (6 connections now open)
2017-12-04T16:45:31.178+0900 I REPL     [rsBackgroundSync] sync source candidate: localhost:37017
2017-12-04T16:45:31.178+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-04T16:45:31.179+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 1ms (1 connections now open to localhost:37017)
2017-12-04T16:45:31.186+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-04T16:45:31.188+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 2ms (2 connections now open to localhost:37017)
2017-12-04T16:45:31.202+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58872 #25 (6 connections now open)
2017-12-04T16:45:31.203+0900 I NETWORK  [conn25] received client metadata from 127.0.0.1:58872 conn25: { driver: { name: "NetworkInterfaceASIO-ShardRegistry", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:31.740+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T16:45:31.741+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T16:45:32.011+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58910 #26 (7 connections now open)
2017-12-04T16:45:32.011+0900 I NETWORK  [conn26] received client metadata from 127.0.0.1:58910 conn26: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.026+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 7 secs, remote host 127.0.0.1:57041)
2017-12-04T16:45:32.026+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58920 #27 (8 connections now open)
2017-12-04T16:45:32.027+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57041 (1 connections now open to localhost:57041 with a 5 second timeout)
2017-12-04T16:45:32.027+0900 I NETWORK  [conn27] received client metadata from 127.0.0.1:58920 conn27: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.028+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58922 #28 (9 connections now open)
2017-12-04T16:45:32.028+0900 I NETWORK  [conn28] received client metadata from 127.0.0.1:58922 conn28: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.028+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58924 #29 (10 connections now open)
2017-12-04T16:45:32.028+0900 I NETWORK  [conn29] received client metadata from 127.0.0.1:58924 conn29: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.029+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37018 (1 connections now open to localhost:37018 with a 5 second timeout)
2017-12-04T16:45:32.030+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47019 (1 connections now open to localhost:47019 with a 5 second timeout)
2017-12-04T16:45:32.030+0900 W NETWORK  [ReplicaSetMonitor-TaskExecutor-0] No primary detected for set s1
2017-12-04T16:45:32.031+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57019 (1 connections now open to localhost:57019 with a 5 second timeout)
2017-12-04T16:45:32.309+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58960 #30 (11 connections now open)
2017-12-04T16:45:32.309+0900 I NETWORK  [conn30] received client metadata from 127.0.0.1:58960 conn30: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.309+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58964 #31 (12 connections now open)
2017-12-04T16:45:32.310+0900 I NETWORK  [conn31] received client metadata from 127.0.0.1:58964 conn31: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.312+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58966 #32 (13 connections now open)
2017-12-04T16:45:32.312+0900 I NETWORK  [conn32] received client metadata from 127.0.0.1:58966 conn32: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.337+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58984 #33 (14 connections now open)
2017-12-04T16:45:32.338+0900 I NETWORK  [conn33] received client metadata from 127.0.0.1:58984 conn33: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.340+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58990 #34 (15 connections now open)
2017-12-04T16:45:32.341+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58994 #35 (16 connections now open)
2017-12-04T16:45:32.341+0900 I NETWORK  [conn34] received client metadata from 127.0.0.1:58990 conn34: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.342+0900 I NETWORK  [conn35] received client metadata from 127.0.0.1:58994 conn35: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:48.933+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:59142 #36 (17 connections now open)
2017-12-04T16:45:48.934+0900 I NETWORK  [conn36] received client metadata from 127.0.0.1:59142 conn36: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:49.058+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:59146 #37 (18 connections now open)
2017-12-04T16:45:49.059+0900 I NETWORK  [conn37] received client metadata from 127.0.0.1:59146 conn37: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:49.455+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:59162 #38 (19 connections now open)
2017-12-04T16:45:49.455+0900 I NETWORK  [conn38] received client metadata from 127.0.0.1:59162 conn38: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:48:01.746+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T16:48:01.748+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-04T16:50:25.205+0900 I -        [conn19] end connection 127.0.0.1:40970 (19 connections now open)
2017-12-04T16:50:25.205+0900 I -        [conn20] end connection 127.0.0.1:40996 (18 connections now open)
2017-12-04T16:50:25.216+0900 I -        [conn22] end connection 127.0.0.1:41006 (17 connections now open)
2017-12-04T16:50:41.223+0900 I -        [conn25] end connection 127.0.0.1:58872 (16 connections now open)
2017-12-04T17:13:31.812+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T17:13:31.814+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T18:02:32.917+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T18:02:32.919+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-04T18:40:03.021+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T18:40:03.022+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-04T19:13:57.195+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 144 secs, remote host 127.0.0.1:47018)
2017-12-04T19:13:57.195+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 144 secs, remote host 127.0.0.1:57017)
2017-12-04T19:13:57.195+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 144 secs, remote host 127.0.0.1:57042)
2017-12-04T19:13:57.216+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:46852 #39 (16 connections now open)
2017-12-04T19:13:57.239+0900 I NETWORK  [conn39] received client metadata from 127.0.0.1:46852 conn39: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T19:13:57.245+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57042 as failed :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T19:13:57.245+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T19:13:59.230+0900 I REPL     [ReplicationExecutor] Member localhost:37019 is now in state PRIMARY
2017-12-04T19:13:59.237+0900 I REPL     [ReplicationExecutor] Member localhost:37017 is now in state SECONDARY
2017-12-04T19:14:20.613+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57042 (1 connections now open to localhost:57042 with a 5 second timeout)
2017-12-04T19:14:20.614+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 23 secs, remote host 127.0.0.1:37017)
2017-12-04T19:14:20.614+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37017 (1 connections now open to localhost:37017 with a 5 second timeout)
2017-12-04T19:14:20.615+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57017 (1 connections now open to localhost:57017 with a 5 second timeout)
2017-12-04T19:14:20.616+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47018 (1 connections now open to localhost:47018 with a 5 second timeout)
2017-12-04T19:14:57.243+0900 I -        [conn39] end connection 127.0.0.1:46852 (16 connections now open)
2017-12-04T19:15:50.304+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T19:15:50.305+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-04T19:42:20.371+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T19:42:20.372+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T19:54:51.085+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T19:54:51.087+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-04T19:54:53.319+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-04T19:54:53.320+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 1ms (3 connections now open to localhost:37017)
2017-12-04T19:55:53.320+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T20:31:51.171+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T20:31:51.175+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 4ms (1 connections now open to localhost:57042)
2017-12-04T20:51:51.222+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T20:51:51.226+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 4ms (1 connections now open to localhost:57040)
2017-12-04T21:15:23.453+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-04T21:15:23.455+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 2ms (3 connections now open to localhost:37017)
2017-12-04T21:16:23.454+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T21:32:51.320+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T21:32:51.321+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T21:49:21.363+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T21:49:21.365+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-04T22:10:51.414+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T22:10:51.416+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-04T22:32:21.474+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T22:32:21.476+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T22:44:51.504+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T22:44:51.505+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T23:05:51.557+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T23:05:51.558+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-05T00:06:21.716+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T00:06:21.720+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 4ms (1 connections now open to localhost:57040)
2017-12-05T00:12:21.734+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T00:12:21.735+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-05T00:36:21.793+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T00:36:21.795+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T00:56:51.843+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T00:56:51.845+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T01:53:51.972+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T01:53:51.974+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T03:05:22.147+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T03:05:22.148+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-05T03:24:52.192+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T03:24:52.194+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-05T03:46:22.247+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T03:46:22.249+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-05T05:09:52.457+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T05:09:52.461+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 4ms (1 connections now open to localhost:57040)
2017-12-05T05:32:52.513+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T05:32:52.515+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-05T05:40:22.533+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T05:40:22.534+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-05T05:43:22.541+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T05:43:22.543+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T05:51:22.559+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T05:51:22.560+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-05T07:58:52.848+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T07:58:52.848+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 0ms (1 connections now open to localhost:57040)
2017-12-05T15:30:53.175+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:53080 #40 (16 connections now open)
2017-12-05T15:30:53.176+0900 I NETWORK  [conn40] received client metadata from 127.0.0.1:53080 conn40: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T15:30:53.178+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 26790 secs, remote host 127.0.0.1:47017)
2017-12-05T15:30:53.231+0900 I REPL     [ReplicationExecutor] Member localhost:37019 is now in state SECONDARY
2017-12-05T15:30:53.245+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57041 as failed :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-05T15:30:53.245+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-05T15:30:55.210+0900 I REPL     [ReplicationExecutor] Member localhost:37017 is now in state PRIMARY
2017-12-05T15:31:09.125+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T15:31:09.128+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 3ms (1 connections now open to localhost:57040)
2017-12-05T15:31:09.645+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:57041)
2017-12-05T15:31:09.648+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57041 (1 connections now open to localhost:57041 with a 5 second timeout)
2017-12-05T15:31:09.650+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:37019)
2017-12-05T15:31:09.653+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37019 (1 connections now open to localhost:37019 with a 5 second timeout)
2017-12-05T15:31:09.654+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:57017)
2017-12-05T15:31:09.657+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57017 (1 connections now open to localhost:57017 with a 5 second timeout)
2017-12-05T15:31:09.661+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47017 (1 connections now open to localhost:47017 with a 5 second timeout)
2017-12-05T15:31:53.192+0900 I -        [conn40] end connection 127.0.0.1:53080 (16 connections now open)
2017-12-05T15:32:39.133+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T15:32:39.134+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-05T15:37:39.146+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T15:37:39.148+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T16:03:27.184+0900 I REPL     [ReplicationExecutor] Starting an election, since we've seen no PRIMARY in the past 10000ms
2017-12-05T16:03:27.184+0900 I REPL     [ReplicationExecutor] conducting a dry run election to see if we could be elected
2017-12-05T16:03:27.184+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:37017
2017-12-05T16:03:27.184+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:37019
2017-12-05T16:03:27.187+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 1518 secs, remote host 127.0.0.1:47019)
2017-12-05T16:03:27.198+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Failed to connect to localhost:37017 - HostUnreachable: End of file
2017-12-05T16:03:27.198+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Dropping all pooled connections to localhost:37017 due to failed operation on a connection
2017-12-05T16:03:27.198+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Failed to close stream: Transport endpoint is not connected
2017-12-05T16:03:27.198+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:37019, took 14ms (2 connections now open to localhost:37019)
2017-12-05T16:03:27.198+0900 I REPL     [ReplicationExecutor] VoteRequester(term 5 dry run) failed to receive response from localhost:37017: HostUnreachable: End of file
2017-12-05T16:03:27.200+0900 I REPL     [ReplicationExecutor] VoteRequester(term 5 dry run) received a yes vote from localhost:37019; response message: { term: 5, voteGranted: true, reason: "", ok: 1.0 }
2017-12-05T16:03:27.200+0900 I REPL     [ReplicationExecutor] dry election run succeeded, running for election
2017-12-05T16:03:27.201+0900 I REPL     [ReplicationExecutor] Member localhost:37017 is now in state SECONDARY
2017-12-05T16:03:27.204+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:37017
2017-12-05T16:03:27.206+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:37017, took 2ms (1 connections now open to localhost:37017)
2017-12-05T16:03:27.206+0900 I REPL     [ReplicationExecutor] VoteRequester(term 6) received a yes vote from localhost:37019; response message: { term: 6, voteGranted: true, reason: "", ok: 1.0 }
2017-12-05T16:03:27.206+0900 I REPL     [ReplicationExecutor] election succeeded, assuming primary role in term 6
2017-12-05T16:03:27.207+0900 I REPL     [ReplicationExecutor] transition to PRIMARY
2017-12-05T16:03:27.207+0900 I REPL     [ReplicationExecutor] Entering primary catch-up mode.
2017-12-05T16:03:27.207+0900 I REPL     [ReplicationExecutor] Heartbeats updated catchup target optime to { ts: Timestamp 1512455892000|1, t: 5 }
2017-12-05T16:03:27.234+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57040 as failed :: caused by :: NotMaster: not master
2017-12-05T16:03:27.234+0900 I NETWORK  [replSetDistLockPinger] Detected bad connection created at 1512369730587282 microSec, clearing pool for localhost:57040 of 0 connections
2017-12-05T16:03:27.234+0900 I NETWORK  [replSetDistLockPinger] Dropping all pooled connections to localhost:57040(with timeout of 5 seconds)
2017-12-05T16:03:27.235+0900 I NETWORK  [replSetDistLockPinger] Ending connection to host localhost:57040(with timeout of 5 seconds) due to bad connection status; 0 connections to that host remain open
2017-12-05T16:03:27.235+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-05T16:03:27.235+0900 W REPL     [rsBackgroundSync] Fetcher stopped querying remote oplog with error: InvalidSyncSource: sync source localhost:37017 (config version: 1; last applied optime: { ts: Timestamp 1512455892000|1, t: 5 }; sync source index: -1; primary index: -1) is no longer valid
2017-12-05T16:03:27.239+0900 I REPL     [rsBackgroundSync] sync source candidate: localhost:37017
2017-12-05T16:03:27.240+0900 I REPL     [rsSync] Caught up to the latest known optime successfully after becoming primary.
2017-12-05T16:03:27.240+0900 I REPL     [rsSync] Exited primary catch-up mode.
2017-12-05T16:03:27.243+0900 I REPL     [rsBackgroundSync] Replication producer stopped after oplog fetcher finished returning a batch from our sync source.  Abandoning this batch of oplog entries and re-evaluating our sync source.
2017-12-05T16:03:27.669+0900 I REPL     [SyncSourceFeedback] SyncSourceFeedback error sending update to localhost:37017: InvalidSyncSource: Sync source was cleared. Was localhost:37017
2017-12-05T16:03:27.737+0900 I NETWORK  [replSetDistLockPinger] Successfully connected to localhost:57040 (1 connections now open to localhost:57040 with a 5 second timeout)
2017-12-05T16:03:27.737+0900 W NETWORK  [replSetDistLockPinger] No primary detected for set csReplSet
2017-12-05T16:03:27.737+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: FailedToSatisfyReadPreference: could not find host matching read preference { mode: "primary" } for set csReplSet
2017-12-05T16:03:28.240+0900 I REPL     [rsSync] transition to primary complete; database writes are now permitted
2017-12-05T16:03:33.185+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56970 #41 (16 connections now open)
2017-12-05T16:03:33.185+0900 I NETWORK  [conn41] received client metadata from 127.0.0.1:56970 conn41: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:03:33.188+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56972 #42 (17 connections now open)
2017-12-05T16:03:33.189+0900 I NETWORK  [conn42] received client metadata from 127.0.0.1:56972 conn42: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:03:33.256+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56986 #43 (18 connections now open)
2017-12-05T16:03:33.256+0900 I NETWORK  [conn43] received client metadata from 127.0.0.1:56986 conn43: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:03:33.258+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56988 #44 (19 connections now open)
2017-12-05T16:03:33.259+0900 I NETWORK  [conn44] received client metadata from 127.0.0.1:56988 conn44: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:03:49.270+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:57148 #45 (20 connections now open)
2017-12-05T16:03:49.271+0900 I NETWORK  [conn45] received client metadata from 127.0.0.1:57148 conn45: { driver: { name: "NetworkInterfaceASIO-ShardRegistry", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T16:03:52.146+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 25 secs, remote host 127.0.0.1:37017)
2017-12-05T16:03:52.147+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37017 (1 connections now open to localhost:37017 with a 5 second timeout)
2017-12-05T16:03:52.150+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 25 secs, remote host 127.0.0.1:57018)
2017-12-05T16:03:52.152+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57018 (1 connections now open to localhost:57018 with a 5 second timeout)
2017-12-05T16:03:52.157+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47019 (1 connections now open to localhost:47019 with a 5 second timeout)
2017-12-05T16:03:57.737+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T16:03:57.739+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T16:04:27.198+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending idle connection to host localhost:37019 because the pool meets constraints; 1 connections to that host remain open
2017-12-05T16:04:27.245+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 1 connections to that host remain open
2017-12-05T16:05:22.904+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-05T16:05:22.906+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-05T16:09:52.916+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T16:09:52.918+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-05T16:13:22.926+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-05T16:13:22.928+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-05T17:08:56.175+0900 I COMMAND  [conn44] command local.oplog.rs command: getMore { getMore: 35297075383, collection: "oplog.rs", maxTimeMS: 5000, term: 6, lastKnownCommittedOpTime: { ts: Timestamp 1512458978000|1, t: 6 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512455892000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 6, readConcern: { afterOpTime: { ts: Timestamp 1512455892000|1, t: 5 } } } planSummary: COLLSCAN cursorid:35297075383 keysExamined:0 docsExamined:0 numYields:1 nreturned:0 reslen:563 locks:{ Global: { acquireCount: { r: 6 } }, Database: { acquireCount: { r: 3 } }, oplog: { acquireCount: { r: 3 } } } protocol:op_command 2352876ms
2017-12-05T17:08:56.175+0900 I COMMAND  [conn42] command local.oplog.rs command: getMore { getMore: 36411012167, collection: "oplog.rs", maxTimeMS: 5000, term: 6, lastKnownCommittedOpTime: { ts: Timestamp 1512458978000|1, t: 6 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512455892000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 6, readConcern: { afterOpTime: { ts: Timestamp 1512455892000|1, t: 5 } } } planSummary: COLLSCAN cursorid:36411012167 keysExamined:0 docsExamined:0 numYields:1 nreturned:0 reslen:563 locks:{ Global: { acquireCount: { r: 6 } }, Database: { acquireCount: { r: 3 } }, oplog: { acquireCount: { r: 3 } } } protocol:op_command 2352876ms
2017-12-05T17:08:56.222+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57042 as failed :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-05T17:08:56.222+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-05T17:09:01.248+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:40976 #46 (21 connections now open)
2017-12-05T17:09:01.248+0900 I NETWORK  [conn46] received client metadata from 127.0.0.1:40976 conn46: { driver: { name: "NetworkInterfaceASIO-ShardRegistry", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:09:01.482+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T17:09:01.483+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-05T17:09:02.022+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 6 secs, remote host 127.0.0.1:57042)
2017-12-05T17:09:02.031+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57042 (1 connections now open to localhost:57042 with a 5 second timeout)
2017-12-05T17:09:02.032+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 6 secs, remote host 127.0.0.1:57017)
2017-12-05T17:09:02.033+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57017 (1 connections now open to localhost:57017 with a 5 second timeout)
2017-12-05T17:09:02.033+0900 W NETWORK  [ReplicaSetMonitor-TaskExecutor-0] No primary detected for set s2
2017-12-05T17:09:02.033+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 6 secs, remote host 127.0.0.1:47018)
2017-12-05T17:09:02.034+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47018 (1 connections now open to localhost:47018 with a 5 second timeout)
2017-12-05T17:09:02.034+0900 W NETWORK  [ReplicaSetMonitor-TaskExecutor-0] No primary detected for set s1
2017-12-05T17:09:10.468+0900 I -        [conn45] end connection 127.0.0.1:57148 (21 connections now open)
2017-12-05T17:09:26.222+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-05T17:09:26.224+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-05T17:19:31.507+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T17:19:31.509+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 3ms (1 connections now open to localhost:57040)
2017-12-05T17:19:41.270+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:41860 #47 (21 connections now open)
2017-12-05T17:19:41.270+0900 I NETWORK  [conn47] received client metadata from 127.0.0.1:41860 conn47: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T17:20:41.270+0900 I -        [conn44] end connection 127.0.0.1:56988 (21 connections now open)
2017-12-05T17:26:31.523+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T17:26:31.524+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-05T17:58:32.228+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T17:58:32.232+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 4ms (1 connections now open to localhost:57040)
2017-12-05T18:32:02.309+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T18:32:02.312+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 3ms (1 connections now open to localhost:57042)
2017-12-05T19:06:02.393+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T19:06:02.394+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-05T19:08:41.453+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:33716 #48 (21 connections now open)
2017-12-05T19:08:41.453+0900 I NETWORK  [conn48] received client metadata from 127.0.0.1:33716 conn48: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T19:09:41.453+0900 I -        [conn42] end connection 127.0.0.1:56972 (21 connections now open)
2017-12-05T19:13:02.408+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T19:13:02.410+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-05T19:20:21.477+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:39352 #49 (21 connections now open)
2017-12-05T19:20:21.479+0900 I NETWORK  [conn49] received client metadata from 127.0.0.1:39352 conn49: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T19:21:21.479+0900 I -        [conn41] end connection 127.0.0.1:56970 (21 connections now open)
2017-12-05T19:49:32.499+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T19:49:32.503+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 3ms (1 connections now open to localhost:57042)
2017-12-05T20:15:01.573+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:50944 #50 (21 connections now open)
2017-12-05T20:15:01.574+0900 I NETWORK  [conn50] received client metadata from 127.0.0.1:50944 conn50: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T20:16:01.572+0900 I -        [conn48] end connection 127.0.0.1:33716 (21 connections now open)
2017-12-05T20:41:32.621+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T20:41:32.623+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T20:46:02.632+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T20:46:02.636+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 4ms (1 connections now open to localhost:57040)
2017-12-05T21:09:32.689+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T21:09:32.691+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T21:34:02.748+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T21:34:02.749+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-05T21:54:02.801+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-05T21:54:02.802+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-05T22:06:02.825+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T22:06:02.827+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T22:11:01.760+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:53248 #51 (21 connections now open)
2017-12-05T22:11:01.761+0900 I NETWORK  [conn51] received client metadata from 127.0.0.1:53248 conn51: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T22:12:01.761+0900 I -        [conn50] end connection 127.0.0.1:50944 (21 connections now open)
2017-12-05T22:48:32.917+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T22:48:32.921+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 4ms (1 connections now open to localhost:57042)
2017-12-05T22:54:02.936+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T22:54:02.937+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-05T23:18:02.992+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T23:18:02.994+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T23:34:33.042+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T23:34:33.044+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-05T23:40:11.926+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:53996 #52 (21 connections now open)
2017-12-05T23:40:11.926+0900 I NETWORK  [conn52] received client metadata from 127.0.0.1:53996 conn52: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-05T23:41:11.927+0900 I -        [conn43] end connection 127.0.0.1:56986 (21 connections now open)
2017-12-05T23:52:33.076+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-05T23:52:33.077+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-06T00:08:21.980+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:54114 #53 (21 connections now open)
2017-12-06T00:08:21.981+0900 I NETWORK  [conn53] received client metadata from 127.0.0.1:54114 conn53: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T00:09:03.116+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-06T00:09:03.118+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-06T00:09:21.981+0900 I -        [conn52] end connection 127.0.0.1:53996 (21 connections now open)
2017-12-06T00:21:33.146+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T00:21:33.147+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-06T00:29:12.043+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:54238 #54 (21 connections now open)
2017-12-06T00:29:12.043+0900 I NETWORK  [conn54] received client metadata from 127.0.0.1:54238 conn54: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T00:30:12.043+0900 I -        [conn53] end connection 127.0.0.1:54114 (21 connections now open)
2017-12-06T01:03:33.251+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T01:03:33.253+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-06T01:50:03.370+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T01:50:03.374+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 4ms (1 connections now open to localhost:57042)
2017-12-06T02:11:33.428+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T02:11:33.430+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (2 connections now open to localhost:57041)
2017-12-06T02:12:33.431+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Ending idle connection to host localhost:57041 because the pool meets constraints; 1 connections to that host remain open
2017-12-06T02:18:33.448+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-06T02:18:33.450+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-06T02:25:03.465+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-06T02:25:03.466+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-06T02:50:03.527+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T02:50:03.528+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-06T03:16:33.588+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T03:16:33.589+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-06T03:34:03.633+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T03:34:03.634+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-06T04:13:22.460+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:55406 #55 (21 connections now open)
2017-12-06T04:13:22.460+0900 I NETWORK  [conn55] received client metadata from 127.0.0.1:55406 conn55: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T04:14:22.460+0900 I -        [conn49] end connection 127.0.0.1:39352 (21 connections now open)
2017-12-06T04:23:33.769+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-06T04:23:33.770+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-06T04:32:03.788+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-06T04:32:03.789+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (1 connections now open to localhost:57040)
2017-12-06T05:04:32.567+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:55650 #56 (21 connections now open)
2017-12-06T05:04:32.568+0900 I NETWORK  [conn56] received client metadata from 127.0.0.1:55650 conn56: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T05:05:32.568+0900 I -        [conn55] end connection 127.0.0.1:55406 (21 connections now open)
2017-12-06T05:09:12.576+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:55664 #57 (21 connections now open)
2017-12-06T05:09:12.576+0900 I NETWORK  [conn57] received client metadata from 127.0.0.1:55664 conn57: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T05:10:12.577+0900 I -        [conn56] end connection 127.0.0.1:55650 (21 connections now open)
2017-12-06T05:53:34.014+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T05:53:34.020+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 6ms (1 connections now open to localhost:57042)
2017-12-06T06:17:34.080+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T06:17:34.081+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-06T06:38:34.135+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T06:38:34.137+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-06T06:51:34.172+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-06T06:51:34.176+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 4ms (1 connections now open to localhost:57040)
2017-12-06T07:49:34.320+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T07:49:34.321+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-06T07:55:04.334+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-06T07:55:04.336+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-06T18:08:08.183+0900 I COMMAND  [ftdc] serverStatus was very slow: { after basic: 0, after asserts: 0, after backgroundFlushing: 0, after connections: 0, after dur: 0, after extra_info: 0, after globalLock: 0, after locks: 0, after network: 0, after opLatencies: 0, after opcounters: 0, after opcountersRepl: 0, after repl: 0, after security: 0, after storageEngine: 0, after tcmalloc: 0, after wiredTiger: 36183600, at end: 36183600 }
2017-12-06T18:08:08.190+0900 I COMMAND  [conn57] command local.oplog.rs command: getMore { getMore: 36411012167, collection: "oplog.rs", maxTimeMS: 5000, term: 6, lastKnownCommittedOpTime: { ts: Timestamp 1512515097000|1, t: 6 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512455892000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 6, readConcern: { afterOpTime: { ts: Timestamp 1512455892000|1, t: 5 } } } planSummary: COLLSCAN cursorid:36411012167 keysExamined:0 docsExamined:1 numYields:1 nreturned:1 reslen:662 locks:{ Global: { acquireCount: { r: 6 } }, Database: { acquireCount: { r: 3 } }, oplog: { acquireCount: { r: 3 } } } protocol:op_command 36185246ms
2017-12-06T18:08:08.190+0900 I COMMAND  [conn47] command local.oplog.rs command: getMore { getMore: 35297075383, collection: "oplog.rs", maxTimeMS: 5000, term: 6, lastKnownCommittedOpTime: { ts: Timestamp 1512515097000|1, t: 6 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512455892000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 6, readConcern: { afterOpTime: { ts: Timestamp 1512455892000|1, t: 5 } } } planSummary: COLLSCAN cursorid:35297075383 keysExamined:0 docsExamined:1 numYields:1 nreturned:1 reslen:662 locks:{ Global: { acquireCount: { r: 6 } }, Database: { acquireCount: { r: 3 } }, oplog: { acquireCount: { r: 3 } } } protocol:op_command 36185246ms
2017-12-06T18:08:08.206+0900 I REPL     [ReplicationExecutor] Member localhost:37019 is now in state SECONDARY
2017-12-06T18:08:08.216+0900 I -        [conn2] end connection 127.0.0.1:36586 (20 connections now open)
2017-12-06T18:08:08.217+0900 I -        [conn24] end connection 127.0.0.1:58790 (19 connections now open)
2017-12-06T18:08:08.225+0900 I REPL     [conn57] stepping down from primary, because a new term has begun: 7
2017-12-06T18:08:08.225+0900 I REPL     [conn47] stepping down from primary, because a new term has begun: 7
2017-12-06T18:08:08.225+0900 I REPL     [replExecDBWorker-1] transition to SECONDARY
2017-12-06T18:08:08.225+0900 I NETWORK  [replExecDBWorker-1] legacy transport layer closing all connections
2017-12-06T18:08:08.225+0900 I NETWORK  [replExecDBWorker-1] Skip closing connection for connection # 57
2017-12-06T18:08:08.225+0900 I NETWORK  [replExecDBWorker-1] Skip closing connection for connection # 54
2017-12-06T18:08:08.225+0900 I NETWORK  [replExecDBWorker-1] Skip closing connection for connection # 51
2017-12-06T18:08:08.225+0900 I NETWORK  [replExecDBWorker-1] Skip closing connection for connection # 47
2017-12-06T18:08:08.225+0900 I NETWORK  [replExecDBWorker-1] Skip closing connection for connection # 46
2017-12-06T18:08:08.226+0900 I -        [conn38] end connection 127.0.0.1:59162 (18 connections now open)
2017-12-06T18:08:08.226+0900 I -        [conn36] end connection 127.0.0.1:59142 (17 connections now open)
2017-12-06T18:08:08.226+0900 I -        [conn35] end connection 127.0.0.1:58994 (16 connections now open)
2017-12-06T18:08:08.226+0900 I -        [conn34] end connection 127.0.0.1:58990 (15 connections now open)
2017-12-06T18:08:08.226+0900 I -        [conn33] end connection 127.0.0.1:58984 (15 connections now open)
2017-12-06T18:08:08.226+0900 I -        [conn30] end connection 127.0.0.1:58960 (14 connections now open)
2017-12-06T18:08:08.226+0900 I -        [conn29] end connection 127.0.0.1:58924 (13 connections now open)
2017-12-06T18:08:08.226+0900 I -        [conn37] end connection 127.0.0.1:59146 (12 connections now open)
2017-12-06T18:08:08.226+0900 I -        [conn27] end connection 127.0.0.1:58920 (11 connections now open)
2017-12-06T18:08:08.227+0900 I -        [conn26] end connection 127.0.0.1:58910 (9 connections now open)
2017-12-06T18:08:08.227+0900 I -        [conn32] end connection 127.0.0.1:58966 (8 connections now open)
2017-12-06T18:08:08.228+0900 I -        [conn31] end connection 127.0.0.1:58964 (7 connections now open)
2017-12-06T18:08:08.228+0900 I -        [conn28] end connection 127.0.0.1:58922 (6 connections now open)
2017-12-06T18:08:08.236+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56564 #58 (6 connections now open)
2017-12-06T18:08:08.237+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56566 #59 (7 connections now open)
2017-12-06T18:08:08.237+0900 I NETWORK  [conn59] received client metadata from 127.0.0.1:56566 conn59: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:08.239+0900 I NETWORK  [conn58] received client metadata from 127.0.0.1:56564 conn58: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:08.252+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Detected bad connection created at 1512373532028738 microSec, clearing pool for localhost:37018 of 0 connections
2017-12-06T18:08:08.254+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57041 as failed :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-06T18:08:08.254+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-06T18:08:08.256+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Dropping all pooled connections to localhost:37018(with timeout of 5 seconds)
2017-12-06T18:08:08.256+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Ending connection to host localhost:37018(with timeout of 5 seconds) due to bad connection status; 0 connections to that host remain open
2017-12-06T18:08:08.256+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Marking host localhost:37018 as failed :: caused by :: HostUnreachable: network error while attempting to run command 'ismaster' on host 'localhost:37018' 
2017-12-06T18:08:08.261+0900 W NETWORK  [ReplicaSetMonitor-TaskExecutor-0] No primary detected for set s0
2017-12-06T18:08:08.267+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Detected bad connection created at 1512457432150587 microSec, clearing pool for localhost:57018 of 0 connections
2017-12-06T18:08:08.267+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Dropping all pooled connections to localhost:57018(with timeout of 5 seconds)
2017-12-06T18:08:08.267+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Ending connection to host localhost:57018(with timeout of 5 seconds) due to bad connection status; 0 connections to that host remain open
2017-12-06T18:08:08.267+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Marking host localhost:57018 as failed :: caused by :: HostUnreachable: network error while attempting to run command 'ismaster' on host 'localhost:57018' 
2017-12-06T18:08:08.268+0900 W NETWORK  [ReplicaSetMonitor-TaskExecutor-0] No primary detected for set s2
2017-12-06T18:08:08.268+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Detected bad connection created at 1512461342033697 microSec, clearing pool for localhost:47018 of 0 connections
2017-12-06T18:08:08.268+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Dropping all pooled connections to localhost:47018(with timeout of 5 seconds)
2017-12-06T18:08:08.268+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Ending connection to host localhost:47018(with timeout of 5 seconds) due to bad connection status; 0 connections to that host remain open
2017-12-06T18:08:08.268+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Marking host localhost:47018 as failed :: caused by :: HostUnreachable: network error while attempting to run command 'ismaster' on host 'localhost:47018' 
2017-12-06T18:08:08.270+0900 W NETWORK  [ReplicaSetMonitor-TaskExecutor-0] No primary detected for set s1
2017-12-06T18:08:09.174+0900 I REPL     [rsBackgroundSync] could not find member to sync from
2017-12-06T18:08:19.177+0900 I REPL     [ReplicationExecutor] Member localhost:37017 is now in state PRIMARY
2017-12-06T18:08:20.736+0900 I -        [conn46] end connection 127.0.0.1:40976 (7 connections now open)
2017-12-06T18:08:24.178+0900 I REPL     [rsBackgroundSync] sync source candidate: localhost:37017
2017-12-06T18:08:24.178+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-06T18:08:24.178+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 0ms (1 connections now open to localhost:37017)
2017-12-06T18:08:24.179+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-06T18:08:24.179+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 0ms (2 connections now open to localhost:37017)
2017-12-06T18:08:24.267+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56636 #60 (7 connections now open)
2017-12-06T18:08:24.267+0900 I NETWORK  [conn60] received client metadata from 127.0.0.1:56636 conn60: { driver: { name: "NetworkInterfaceASIO-ShardRegistry", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:24.444+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56650 #61 (8 connections now open)
2017-12-06T18:08:24.444+0900 I NETWORK  [conn61] received client metadata from 127.0.0.1:56650 conn61: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:24.521+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56654 #62 (9 connections now open)
2017-12-06T18:08:24.521+0900 I NETWORK  [conn62] received client metadata from 127.0.0.1:56654 conn62: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:25.100+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56662 #63 (10 connections now open)
2017-12-06T18:08:25.100+0900 I NETWORK  [conn63] received client metadata from 127.0.0.1:56662 conn63: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:36.208+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T18:08:36.208+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 0ms (1 connections now open to localhost:57042)
2017-12-06T18:08:38.181+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56754 #64 (11 connections now open)
2017-12-06T18:08:38.181+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56756 #65 (12 connections now open)
2017-12-06T18:08:38.181+0900 I NETWORK  [conn64] received client metadata from 127.0.0.1:56754 conn64: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.181+0900 I NETWORK  [conn65] received client metadata from 127.0.0.1:56756 conn65: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.206+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56764 #66 (13 connections now open)
2017-12-06T18:08:38.206+0900 I NETWORK  [conn66] received client metadata from 127.0.0.1:56764 conn66: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.217+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56772 #67 (14 connections now open)
2017-12-06T18:08:38.217+0900 I NETWORK  [conn67] received client metadata from 127.0.0.1:56772 conn67: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.225+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56776 #68 (15 connections now open)
2017-12-06T18:08:38.225+0900 I NETWORK  [conn68] received client metadata from 127.0.0.1:56776 conn68: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.240+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 30 secs, remote host 127.0.0.1:57041)
2017-12-06T18:08:38.241+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57041 (1 connections now open to localhost:57041 with a 5 second timeout)
2017-12-06T18:08:38.255+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-06T18:08:38.255+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 0ms (1 connections now open to localhost:57040)
2017-12-06T18:08:38.262+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56794 #69 (16 connections now open)
2017-12-06T18:08:38.263+0900 I NETWORK  [conn69] received client metadata from 127.0.0.1:56794 conn69: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.267+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56796 #70 (17 connections now open)
2017-12-06T18:08:38.268+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56802 #71 (18 connections now open)
2017-12-06T18:08:38.268+0900 I NETWORK  [conn71] received client metadata from 127.0.0.1:56802 conn71: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.269+0900 I NETWORK  [conn70] received client metadata from 127.0.0.1:56796 conn70: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.271+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56804 #72 (19 connections now open)
2017-12-06T18:08:38.271+0900 I NETWORK  [conn72] received client metadata from 127.0.0.1:56804 conn72: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.271+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37018 (1 connections now open to localhost:37018 with a 5 second timeout)
2017-12-06T18:08:38.271+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:56808 #73 (20 connections now open)
2017-12-06T18:08:38.272+0900 I NETWORK  [conn73] received client metadata from 127.0.0.1:56808 conn73: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-06T18:08:38.274+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57018 (1 connections now open to localhost:57018 with a 5 second timeout)
2017-12-06T18:08:38.279+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47018 (1 connections now open to localhost:47018 with a 5 second timeout)
2017-12-06T18:09:08.240+0900 I -        [conn51] end connection 127.0.0.1:53248 (20 connections now open)
2017-12-06T18:09:08.250+0900 I -        [conn54] end connection 127.0.0.1:54238 (19 connections now open)
2017-12-06T18:13:08.262+0900 I -        [conn47] end connection 127.0.0.1:41860 (18 connections now open)
2017-12-06T18:13:08.262+0900 I -        [conn57] end connection 127.0.0.1:55664 (18 connections now open)
2017-12-06T18:13:34.283+0900 I -        [conn60] end connection 127.0.0.1:56636 (16 connections now open)
2017-12-06T18:30:36.266+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T18:30:36.267+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-06T18:54:06.756+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T18:54:06.756+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 0ms (1 connections now open to localhost:57042)
2017-12-06T18:59:36.763+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T18:59:36.764+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-06T19:14:36.787+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T19:14:36.787+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-06T19:22:06.804+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T19:22:06.806+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-06T19:57:36.886+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T19:57:36.890+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 4ms (1 connections now open to localhost:57041)
2017-12-06T21:04:05.569+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-06T21:04:05.572+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 3ms (3 connections now open to localhost:37017)
2017-12-06T21:05:05.572+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 2 connections to that host remain open
2017-12-06T21:39:07.148+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T21:39:07.154+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 6ms (1 connections now open to localhost:57042)
2017-12-06T21:45:07.169+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T21:45:07.173+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 4ms (1 connections now open to localhost:57042)
2017-12-06T21:56:07.198+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T21:56:07.200+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-06T22:06:37.225+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-06T22:06:37.226+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-06T22:43:37.318+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T22:43:37.322+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 3ms (1 connections now open to localhost:57042)
2017-12-06T23:00:07.366+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T23:00:07.368+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-06T23:48:37.479+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-06T23:48:37.481+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-07T00:23:37.561+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-07T00:23:37.561+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 0ms (1 connections now open to localhost:57042)
2017-12-07T00:56:37.638+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-07T00:56:37.639+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-07T01:11:37.677+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-07T01:11:37.680+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 3ms (1 connections now open to localhost:57041)
2017-12-07T01:41:37.755+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-07T01:41:37.756+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-07T02:51:07.936+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-07T02:51:07.937+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-07T03:22:26.295+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-07T03:22:26.297+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 2ms (3 connections now open to localhost:37017)
2017-12-07T03:23:26.296+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 2 connections to that host remain open
2017-12-07T03:23:38.034+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-07T03:23:38.035+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-07T05:21:06.542+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-07T05:21:06.544+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 2ms (3 connections now open to localhost:37017)
2017-12-07T05:22:06.543+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 2 connections to that host remain open
2017-12-07T05:36:08.365+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-07T05:36:08.366+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-07T05:49:08.398+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-07T05:49:08.401+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 3ms (1 connections now open to localhost:57042)
2017-12-07T06:07:08.448+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-07T06:07:08.449+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-07T06:14:08.467+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-07T06:14:08.468+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-07T06:19:38.483+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-07T06:19:38.484+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-07T06:36:38.527+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-07T06:36:38.528+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-07T06:47:38.555+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-07T06:47:38.557+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
